docs/tutorials/hello_many_worlds.ipynb

```diff
@@ -915,6 +917,72 @@
         "From this plot you can see that the neural network has learned to overcome the systematic miscalibration."
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "C2RfWismj66S",
+        "colab_type": "text"
+      },
+      "source": [
+        "### 2.6 Verifying outputs\n",
+        "Now that you have trained this model, you can use it to correct your system that has this calibration error. Using Cirq you can do this with:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "RoIlb7r7j5SY",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "def check_error(command_values, desired_values):\n",
+        "  \"\"\"Based on the value in `command_value` see how well you could prepare\n",
+        "  the full circuit to have `desired_value` when taking expectation w.r.t. Z.\"\"\"\n",
+        "  params_to_prepare_output = controller(command_values).numpy()\n",
+        "  full_circuit = noisy_preparation + model_circuit\n",
+        "\n",
+        "  # Test how well you can prepare a state to get expectation the expectation\n",
+        "  # value in `desired_values`\n",
+        "  for index in [0, 1]:\n",
+        "    state = cirq_simulator.simulate(\n",
+        "        full_circuit,\n",
+        "        {s:v for (s,v) in zip(control_params, params_to_prepare_output[index])}\n",
+        "    ).final_state\n",
+        "    expectation = z0.expectation_from_wavefunction(state, {qubit: 0}).real\n",
+        "    print(f'For a desired output (expectation) of {desired_values[index]} with'\n",
+        "          f' noisy preparation, the controller\\nnetwork found the following '\n",
+        "          f'values for theta: {params_to_prepare_output[index]}\\nWhich gives an'\n",
+        "          f' actual expectation of: {expectation}\\n')\n",
+        "\n",
```

Comment:
perhaps add another newline here to make the following call stand out. Or move the following to a separate cell

Reply:
Added newline.